packages/esm-ward-app/src/beds/occupied-bed.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { type Patient } from '@openmrs/esm-framework';
+import { type Bed } from '../types';
+import styles from './occupied-bed.scss';
+import { Tag } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import WardPatientCard from '../ward-patient-card/ward-pateint-card.component';
+
+export interface OccupiedBedProps {
+  patients: Patient[];
+  bed: Bed;
+}
+const OccupiedBed: React.FC<OccupiedBedProps> = ({ patients, bed }) => {
+  return (
+    <div className={styles.occupiedBed}>
+      {patients.map((patient, index: number) => {
+        const last = index === patients.length - 1;
+        return (
+          <div key={patient.uuid + ' ' + index}>
+            <WardPatientCard patient={patient} bed={bed} status={'admitted'} />
+            {!last && <BedShareDivider />}
+          </div>
+        );
+      })}
+    </div>
+  );
+};
+
+const BedShareDivider = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.bedDivider}>
+      <div className={styles.bedDividerLine}></div>
+      <Tag>{t('bedshare', 'Bed share')}</Tag>
```

Comment:
Not entirely sure I follow what this corresponds to in the designs?

Reply:
@ibacher see images in Zeppelin, also summarized here:  https://openmrs.atlassian.net/browse/O3-3198

Reply:
Is this a placeholder tag? I was looking at the images and not really getting what from those images this implements.

Reply:
It's this divider for when there are multiple patients assigned to the same bed:

![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/509602/3f6de993-3b45-4dc8-b918-76d348b7e0f6)

Reply:
Got it! I missed that one. Thanks!